src/hacl.rs

```diff
@@ -13,7 +13,7 @@ pub(crate) mod aesgcm;
 pub(crate) mod blake2;
 pub(crate) mod chacha20_poly1305;
 pub(crate) mod curve25519;
-#[cfg(not(target_arch ="wasm32"))]
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Was this a manual change or automatically formatted by cargo? Running `cargo fmt` on the dev branch does not give me this diff.

Reply:
That's `cargo fmt`. I added it to CI.